<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogImageGrabberSettings</class>
 <widget class="QDialog" name="DialogImageGrabberSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="sourceLabel">
     <property name="text">
      <string>Source</string>
     </property>
     <property name="buddy">
      <cstring>sourceLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="sourceLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="fPSLabel">
     <property name="text">
      <string>FPS:</string>
     </property>
     <property name="buddy">
      <cstring>fPSSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="fPSSpinBox"/>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="checkBoxNoLimit">
     <property name="text">
      <string>Unlimited</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="labelError">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pushButtonStart">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogImageGrabberSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogImageGrabberSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
